### PR TITLE
Hunt improvement

### PR DESCRIPTION
**Pull request by DziubanMaciej**

### CODE REVIEW

File: ConsoleApp1/Ecosystem/Biotic/Animal.cs [hunk 1]
```diff
@@ -94,6 +94,25 @@ virtual public void StayWithAlpha()
             Follow(myAlpha.x, myAlpha.y);
         }
 
+        protected Plant FindClosestPlant(World world) {
+            double closestPreyDistance = world.sizeX * world.sizeY;
+            Plant closestPrey = null;
+            for (int j = 0; j < world.plants.Count(); j++) {
+                Plant prey = world.plants[j];  // TODO: out of range exception?
```
Comment (Zarazinski) [inline ConsoleApp1/Ecosystem/Biotic/Animal.cs:101]: I would vote for using a foreach loop here. We don't need the "j" counter and we will avoid possible "out of range" exception.

Comment (Zarazinski) [reply ConsoleApp1/Ecosystem/Biotic/Animal.cs:101]: `foreach (Plant prey : world.plants)   {/*...*/}`

**Review by Zarazinski — APPROVED** [review]
Really good work! :) I've added one suggestion in comments below.

**Review by Lauralicja — APPROVED** [review]

**Comment by Lauralicja** [comment]
Wow thank you!

**Review by Lauralicja — COMMENTED** [review]
:)

**Pull request merged**